### PR TITLE
#21726 — fix orgunit

### PR DESCRIPTION
**Pull request by danchr**
This change adjusts the org. unit API to better correspond to the wishes of the frontenders.

### CODE REVIEW

File: mora/service/orgunit.py [hunk 1]
```diff
@@ -63,23 +100,26 @@ def get_one_orgunit(c, unitid, unit=None,
...
     elif details is UnitDetails.NCHILDREN:
+        r.update(base_info)
+
         children = c.organisationenhed(overordnet=unitid, gyldighed='Aktiv')
 
         r['child_count'] = len(children)
 
-    else:
+    elif details is UnitDetails.FULL:
+        r[keys.ORG_UNIT] = base_info
```
Comment (cmoesgaard) [inline mora/service/orgunit.py]: It needs to be clear that these are properties of the object itself, and not relations to other objects.

Comment (cmoesgaard) [reply mora/service/orgunit.py]: Having the objects properties nested one level down is also inconsistent with how every other detail object is structured.

Comment (danchr) [reply mora/service/orgunit.py]: Really? It's an explicit request from Anders…

Comment (danchr) [reply mora/service/orgunit.py]: (And I agree about your first point — I just wasn't sure whether it affected other objects as well.)

Comment (danchr) [reply mora/service/orgunit.py]: Agreed.

**Pull request closed without merging**